communications/newsletters/newsletter_11_Feb2020.md

```diff
@@ -0,0 +1,198 @@
+These members are (listed alphabetically): [Adina Wagner](https://twitter.com/AdinaKrik), [Alex Chan](https://twitter.com/alexwlchan), [Anna Hadjitofi](https://www.turing.ac.uk/people/researchers/anna-hadjitofi), [Arielle Bennett](https://uk.linkedin.com/in/ariellebennett), [Barbara Vreede](https://twitter.com/barbaravreede), [Christina Hitrova](https://twitter.com/C_Hitrova), [Esther Plomp](https://twitter.com/PhDToothFAIRy), [Frances Madden](https://twitter.com/maddenfc), [Heidi Seibold](https://twitter.com/HeidiBaya), [Kesson Magid](https://twitter.com/kesson), [Martina G. Vilas](https://twitter.com/martinagvilas), [Mateusz Kuzak](https://twitter.com/matkuzak), [Nathan Begbie](https://www.linkedin.com/in/nathan-begbie-161902173/), [Nicolás Alessandroni](https://twitter.com/nalessandroni), and [Tony Yang](https://twitter.com/tony_yzy) 💖 🙌 🚀 🌟
+
+One of the past attendees, [Jade Pickering](https://twitter.com/Jade_Pickering) will be returning as a helper along with the core contributors [Louise Bowler](https://www.turing.ac.uk/people/researchers/louise-bowler) and [Sarah Gibson](https://twitter.com/drsarahlgibson).
+Those who are not located in London have also received travel and accommodation support from *The Turing Way* project.
+We are looking forward to welcoming them and working with them in person!
+
+#### Your comments are needed on the rubric for selection
+
+A call for application was opened in December 2019, inviting applications from the interested candidates sharing their ideas for contribution to *The Turing Way* project.
+They were also asked to justify how they will benefit from their participation at this event.
+We adapted the Mozilla Open Leadership selection rubric for ranking and scoring all the applications.
+This rubric is currently available in [pull request #804](https://github.com/alan-turing-institute/the-turing-way/pull/804).
+We welcome your comments and suggestions on this rubric for the selection process in our future events.
+
+### Translating *The Turing Way* into additional languages
+
+Ang Li and Yini He have translated four initial chapters of *The Turing Way* in Chinese. 
+These researchers from China are spending a year in The Alan Turing Institute in the [Whitaker Lab](https://twitter.com/Whitaker_Lab) and are working on different topics in the neurobiology field while contributing to *The Turing Way* project.
+
+![Tweet from Turing Way project and Kirstie Whitaker celebrating Tony, Ang & Yini's progress translating the book](/communications/newsletters/images/Tweet-Translation-Feb2020.png)
+*Read the [full thread on Twitter](https://twitter.com/turingway/status/1225719523497697280)*
+
+Tony Yang, who is conducting his PhD research at Imperial College London has set out to develop standardised guidelines and infrastructure for translation efforts in Chinese and other languages within the community.
```

Comment:
```suggestion
Tony Yang, who is conducting his PhD research at Imperial College London. has set out to develop standardised guidelines and infrastructure for translation efforts in Chinese and other languages within the community.
```

Reply:
Should be a comma not a period. I'll fix locally - thanks for point it out!